src/components/ReportActionItem/ReportPreview.tsx

```diff
@@ -322,7 +322,23 @@ function ReportPreview({
     const shouldShowScanningSubtitle = numberOfScanningReceipts === 1 && numberOfRequests === 1;
     const shouldShowPendingSubtitle = numberOfPendingRequests === 1 && numberOfRequests === 1;
 
+    const isPayAtEndExpense = numberOfRequests === 1 && TransactionUtils.isPayAtEndExpense(allTransactions[0]);
```

Comment:
Should we just use `isPayAtEndExpenseReport` here?
```suggestion
    const isPayAtEndExpense = ReportUtils.isPayAtEndExpenseReport(reportID, allTransactions);
```

Reply:
Totally, now that we can use the current transactions.